docs/websockets.rst

```diff
@@ -306,3 +306,25 @@ When we pull new rules from the remote repository and there are conflicts in the
 
 
 - ``num_of_conflicts``: Number of conflicts found during the processing of accounting rules.
+
+
+Transaction decoding process
+============================
+
+When the endpoint to start the task for decoding undecoded transactions is queried we send ws messages to inform about the progress.
+
+::
+    {
+        "type":"evm_decoding_pending_transactions",
+        "data":{
+            "chain":"ethereum",
```

Comment:
Didn't we say to call this evm_chain for consistency? Check other arguments in the same file. It's `evm_chain`.

```suggestion
            "evm_chain":"ethereum",
```